website/src/components/PreviousExeResponsiveMobile/PreviousExeResMobile.jsx

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, {useState} from 'react';
+import moment from 'moment';
+
+import './previousExeResMobile.css'
+
+const PreviousExeRes = ({data, className}) => {
+
+    const getStatusClass = (status) => {
+        switch (status) {
+            case 'finished':
+              return 'finished';
+            case 'failed':
+              return 'failed';
+            case 'started':
+              return 'started';
+            default:
+              return 'default';
+          }
+    }
+
+
+    // Create a Moment object from the given date string
+    const startedDate = moment(data.started_at)
+    const finishedDate = moment(data.finished_at)
+    
+    // Format the date using the format method of Moment.js
+    // Here, we use 'DD/MM/YYYY HH:mm:ss' as the desired format
+    const formattedStartedDate = startedDate.format('MM/DD/YYYY HH:mm')
+    const formattedFinishedDate = finishedDate.format('MM/DD/YYYY HH:mm')
+
+
+    const [maxHeight, setMaxHeight] = useState(70);
+
+    const handleClick = () => {
+        if (maxHeight === 70) {
+            setMaxHeight(400);
+          } else {
+            setMaxHeight(70);
+          }
+        };
```

Comment:
Something is off with the format here. The closing `}` is not indented properly :)